templates/_linux.yaml

```diff
@@ -16,7 +16,7 @@
       /objects[0].spec.template.spec.networks
 
 {% for osl in oslabels %}
-    name.os.template.kubevirt.io/{{ osl }}: {{ lookup('osinfo', osl).name }}
+    name.os.template.kubevirt.io/{{ osl }}: {{ lookup('osinfo', oslabels[0]).name }} or higher VM
```

Comment:
You should make sure the oslabels are sorted too. Just to make sure oslabels[0] is the oldest one.

Reply:
oslabels are already sorted from previous task (https://github.com/kubevirt/common-templates/blob/master/generate-templates.yaml#L13)